backend/app.hopps.org/src/main/java/app/hopps/org/rest/OrganizationResource.java

```diff
@@ -63,6 +69,29 @@ public Response getOrganizationBySlug(@PathParam("slug") String slug) {
         return Response.ok(organization).build();
     }
 
+    @GET
+    @Path("/my")
+    @Produces(MediaType.APPLICATION_JSON)
+    @Operation(summary = "Get my organization", description = "Retrieves the details of an organization the current user is assigned to.")
+    @APIResponse(responseCode = "200", description = "Own organization retrieved successfully", content = @Content(mediaType = MediaType.APPLICATION_JSON, schema = @Schema(implementation = Organization.class)))
+    @APIResponse(responseCode = "404", description = "Organization not found for user")
+    public Organization getMyOrganization(@Context SecurityContext securityContext) {
+
+        String userName = securityContext.getUserPrincipal().getName();
+        Member me = memberRepository.findByEmail(userName);
+        if (me == null) {
+            throw new NotFoundException("Organization of User not found");
```

Comment:
Wäre gut, wenn wir auch eine Fehlermeldung bekommen statt nur den code. Quarkus ist ja leider so schlau und lässt die Fehlermeldung weg aus der Response.
Gerne mal in Zeile 142 und 143 schauen.